assets/cases/network/NetworkDownload.ts

```diff
@@ -31,16 +31,19 @@ export class NetworkDownload extends Component {
             console.log(bytesReceived, totalBytesReceived, totalBytesExpected)
             let progress = (totalBytesReceived / totalBytesExpected * 100).toFixed(1);
             this.text.string = totalBytesReceived + " ==> " + progress + "%";
+            this.node.emit('onProgress'); // for auto-test
```

Comment:
Why emit event?

Reply:
Because the automation script needs to take a screenshot after the download events.